dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/tracker/export/trackedentity/TrackedEntityServiceTest.java

```diff
@@ -236,13 +236,13 @@ void setUp() {
     userWithSearchInAllAuthority.setTeiSearchOrganisationUnits(
         Set.of(orgUnitA, orgUnitB, orgUnitC));
 
-    teaA = createTrackedEntityAttribute('A', ValueType.TEXT);
-    teaA.setUnique(true);
-    manager.save(teaA, false);
-    TrackedEntityAttribute teaB = createTrackedEntityAttribute('B', ValueType.TEXT);
-    manager.save(teaB, false);
-    teaC = createTrackedEntityAttribute('C', ValueType.TEXT);
-    manager.save(teaC, false);
+    tetaA = createTrackedEntityAttribute('A', ValueType.TEXT);
```

Comment:
Should these variables be called teaA, teaB and pteaC?
Their type is `TrackedEntityAttribute`, not `TrackedEntityTypeAttribute`.